### PR TITLE
AAP-7870 updated installation guide

### PR DESCRIPTION
**Pull request by emurtoug**
Updated [1] [proc-installing-the-aap-setup-bundle.adoc](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_installation_guide/disconnected-installation#installing_the_setup_bundle) and [2] p[roc-synchronizing-rpm-repositories-by-using-reposync.adoc](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_installation_guide/disconnected-installation#proc-synchronizing-rpm-repositories-by-using-reposync_disconnected-installation) within the AAP Installation Guide

[1] Updated command to untar the bundle 
[2] Fixed git merge conflict

### CODE REVIEW

**Review by dcdacosta — APPROVED** [review]
/LGTM

File: downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc [hunk 1]
```diff
@@ -19,8 +19,8 @@ The download and installation of the setup bundle needs to be located on the aut
 +
 ----
 $ tar xvf \
-   ansible-automation-platform-setup-bundle-2.2.1.tar.gz
-$ cd ansible-automation-platform-setup-bundle-2.2.1-1
+   ansible-automation-platform-setup-bundle-2.3.tar.gz
```
Comment (saito-hideki) [inline downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc:22]: Hi,

It seems that the latest version setup-bundle is `ansible-automation-platform-setup-bundle-2.3-1.1.tar.gz`. So you need to relate 2.3 with 2.3-1.1 if you want to use the current setup-bundle version of 2.3 release.

Or you can use an abstract word, such as `ansible-automation-platform-setup-bundle-<RELEASE_VERSION>.tar.gz`. I would recommend this, although the user would need to replace it accordingly.

File: downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc [hunk 2]
```diff
@@ -19,8 +19,8 @@ The download and installation of the setup bundle needs to be located on the aut
...
+$ cd ansible-automation-platform-setup-bundle-2.3-1
```
Comment (saito-hideki) [inline downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc:23]: the same as above.

**Review by saito-hideki — CHANGES_REQUESTED** [review]
Hi! 
Thank you for the PR! I left a comment regarding the version notation. :)

**Review by dcdacosta — APPROVED** [review]
LGTM

**Pull request merged**